packages/ui/src/components/va-button/VaButton.stories.ts

```diff
@@ -1,12 +1,433 @@
-import VaButton from './VaButton.demo.vue'
+import { VaButton } from './'
+import { userEvent } from '../../../.storybook/interaction-utils/userEvent'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaButton',
   component: VaButton,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
+export const Default = () => ({
   components: { VaButton },
-  template: '<VaButton/>',
+  template: `
+    <VaButton>
+      Button
+    </VaButton>
+  `,
+})
+
+export const Color = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton color="warning">
+      Button
+    </VaButton>
+  `,
+})
+
+export const Gradient = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton gradient>
+      Button
+    </VaButton>
+  `,
+})
+
+export const BackgroundOpacity = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton :backgroundOpacity="0.25">
+      Button
+    </VaButton>
+  `,
+})
+
+export const TextColor = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton textColor="warning">
+      Button
+    </VaButton>
+  `,
+})
+
+export const TextOpacity = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton :textOpacity="0.75">
+      Button
+    </VaButton>
+  `,
+})
+
+export const BorderColor = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton borderColor="warning">
+      Button
+    </VaButton>
+  `,
+})
+
+export const Disabled = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton disabled>
+      Button
+    </VaButton>
+  `,
+})
+
+export const Plain = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton plain>
+      Button
+    </VaButton>
+  `,
+})
+
+export const Round = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton round>
+      Button
+    </VaButton>
+    &nbsp;
+    <VaButton round icon="warning"/>
+  `,
+})
+
+export const Block = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton block>
+      Button
+    </VaButton>
+  `,
+})
+
+export const Icon = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton icon="va-warning">
+      Button
+    </VaButton>
+  `,
+})
+
+export const IconRight = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton iconRight="va-warning">
+      Button
+    </VaButton>
+  `,
+})
+export const IconOnly = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton iconRight="va-warning"/>
+  `,
+})
+
+export const IconColor = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton
+      icon="va-warning"
+      iconColor="warning"
+    >
+      Button
+    </VaButton>
+  `,
+})
+
+export const Loading = () => ({
+  components: { VaButton },
+  template: `
+    <VaButton loading>
+      Button
+    </VaButton>
+  `,
+})
+
+export const Size = () => ({
+  components: { VaButton },
+  template: `
+    [small]
```

Comment:
<img width="565" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/c3ea054d-1ea5-482e-ab69-c787b36b4e46">

Let's add vertical space so that it doesn't look as ugly
```
[small]
[button][button][button]
[medium]
[button][button][button]
etc
``